### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by aokomorowski**
I created polish translation for this awesome guide.

### CODE REVIEW

File: Translation-French.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Français ·
 [Русский](./Translation-Russian.md) ·
 [Deutsch](./Translation-German.md) ·
 [日本語](./Translation-Japanese.md)
```
Comment (i-ky) [inline Translation-French.md]: Add the separator, please (in all translations).

File: Translation-Chinese.md [hunk 2]
```diff
@@ -6,62 +6,64 @@
...
 [日本語](./Translation-Japanese.md)
+[Polski](./Translation-Polish.md)
 
 ---
 
 # 学习 LaTeX 从现在开始
 
 ![](https://upload.wikimedia.org/wikipedia/commons/9/92/LaTeX_logo.svg)
 
-**译者前言：** *本人尽量按照原作者原意翻译，但水平有限，如有翻译不妥之处，欢迎指正。如果您喜欢本文，欢迎分享给更多人看到。*
```
Comment (i-ky) [inline Translation-Chinese.md]: What's happening here? Were you using some sort of WYSIWYG Markdown editor which prefers `_` over `*`?

File: readme.md [hunk 3]
```diff
@@ -5,60 +5,61 @@ English ·
 [Français](./Translation-French.md) ·
 [Русский](./Translation-Russian.md) ·
 [Deutsch](./Translation-German.md) ·
-[日本語](./Translation-Japanese.md)
+[日本語](./Translation-Japanese.md) .
```
Comment (i-ky) [inline readme.md]: You use wrong character as separator, it is *not* a regular dot.

**Review by i-ky — CHANGES_REQUESTED** [review]
Please avoid making unrelated changes, e.g. use a plain text editor for existing translations.

File: Translation-Chinese.md (hunk 2 above)
Comment (aokomorowski) [reply Translation-Chinese.md]: Probably it's effect of VSCode format-on-save feature.

File: Translation-Polish.md [hunk 4]
```diff
@@ -0,0 +1,499 @@
...
+### Spis treści
+
+- [Czym jest Latex?](#czym-jest-latex)
+- [Dlaczego miałbym go używać?](#dlaczego-miałbym-go-używać)
+- [Konfiguracja](#konfiguracja)
+- [Pierwsze kroki](#pierwsze-kroki)
+- [Rzut oka na szczegóły](#Rzut-oka-na-szczegóły)
+- [Wiele języków w jednym dokumencie](#Wiele-języków-w-jednym-dokumencie)
+- [Listy i wypunktowania](#listy-i-wypunktowania)
+- [Akapity i rozdziały](#Ustępy-i-rozdziały)
```
Comment (i-ky) [inline Translation-Polish.md]: Broken link, should be
```suggestion
- [Akapity i rozdziały](##akapity-i-rozdziały)
```

File: Translation-French.md (hunk 1 above)
Comment (aokomorowski) [reply Translation-French.md]: Done

File: Translation-Polish.md [hunk 5]
```diff
@@ -0,0 +1,499 @@
...
+
+<div id='donation'/>
+
+[![](https://www.paypalobjects.com/en_US/i/btn/btn_donate_SM.gif)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=5ZG5Z47L2ZGYC)
+
+A beer in your country can buy a meal in mine.
+
+## Adnotacja od autora tłumaczenia
+
+Autorem tłumaczenia jest Artur Komorowski (aokomorowski@gmail.com). Jeśli widzisz w tekście błędy - proszę, popraw je. Jeśli uznasz, że LaTeX zmienił Twoje życie - podziękuj autorowi.
```
Comment (i-ky) [inline Translation-Polish.md]: I wouldn't recommend leaving your email in public. If you want to, you can leave a link to your GitHub profile.
```suggestion
Autorem tłumaczenia jest Artur Komorowski (@aokomorowski). Jeśli widzisz w tekście błędy - proszę, popraw je. Jeśli uznasz, że LaTeX zmienił Twoje życie - podziękuj autorowi.
```

File: Translation-Chinese.md (hunk 2 above)
Comment (aokomorowski) [reply Translation-Chinese.md]: Restored

File: readme.md (hunk 3 above)
Comment (aokomorowski) [reply readme.md]: Fixed

File: Translation-Polish.md (hunk 4 above)
Comment (aokomorowski) [reply Translation-Polish.md]: Resolved

**Comment by luong-komorebi** [comment]
cc @i-ky 
Since both of you have been working hard on this.  I would like to say thank you, and that I ll get this merged. Any fixes can come later, as I myself found that the translation is valuable and it's worth being taken to the master branch asap.

**Comment by i-ky** [comment]
@luong-komorebi, I've created #60 to fix what I believe is the last issue. The rest of them I believe have been successfully addressed. Good job, @aokomorowski!

**Pull request merged**